base/abstractarray.jl

```diff
@@ -325,15 +325,15 @@ end
 checkbounds(A::AbstractArray) = checkbounds(A, 1) # 0-d case
 
 """
-    checkbounds_indices(Bool, IA, I)
+    checkbounds_indices(Bool, A, IA, I)
```

Comment:
should we deprecate the old signature? DistributedArrays, ImageCore, and ImageFiltering look like they're calling this